fronts-client/src/bundles/__tests__/chefsBundle.spec.ts

```diff
@@ -8,124 +8,161 @@ interface MockedResponse {
+    internalName: 'Anna Del Conte',
+    lastName: 'conteanna',
+    r2ContributorId: '66638',
+    type: 'contributor',
+    webTitle: 'Anna Del Conte',
+    webUrl: 'https://www.theguardian.com/profile/anna-del-conte',
+  },
+  'profile/anna-jones': {
+    apiUrl: 'https://content.guardianapis.com/profile/anna-jones',
+    bio: '',
+    bylineImageUrl: 'https://uploads.guim.co.uk/2018/01/29/Anna-Jones.jpg',
+    bylineLargeImageUrl:
+      'https://uploads.guim.co.uk/2018/01/29/Anna_Jones,_L.png',
+    firstName: 'Anna',
+    id: 'profile/anna-jones',
+    internalName: 'Anna Jones',
+    lastName: 'Jones',
+    r2ContributorId: '64120',
+    type: 'contributor',
+    webTitle: 'Anna Jones',
+    webUrl: 'https://www.theguardian.com/profile/anna-jones',
+  },
+};
+const quickTimeout = () =>
+  new Promise((resolve) => window.setTimeout(resolve, 10));
 
-describe("chefsBundle", ()=>{
-  beforeEach(()=>fetchMock.reset());
+describe('chefsBundle', () => {
+  beforeEach(() => fetchMock.reset());
 
-  it("fetchChefs should fetch chefs by param, look up CAPI details for profile tags, and add them to the state", async ()=>{
+  it('fetchChefs should fetch chefs by param, look up CAPI details for profile tags, and add them to the state', async () => {
     const store = createStoreAndFetchMock([
       {
-        pattern: "https://recipes.guardianapis.com/keywords/contributors?q=anna",
-        response: annaMockRecipe
+        pattern:
+          'https://recipes.guardianapis.com/keywords/contributors?q=anna',
+        response: annaMockRecipe,
       },
       {
-        pattern: "/api/live/tags?type=contributor&ids=profile%2Fanna-jones%2Cprofile%2Fanna-del-conte&show-elements=image&show-fields=all",
-        response: tagLookupResponse
-      }
-      ]);
-    await store.dispatch(fetchChefs({query: "anna"}) as any);
+        pattern:
+          '/api/live/tags?type=contributor&ids=profile%2Fanna-jones%2Cprofile%2Fanna-del-conte&show-elements=image&show-fields=all&page-size=100',
+        response: tagLookupResponse,
+      },
+    ]);
+    await store.dispatch(fetchChefs({ query: 'anna' }) as any);
     await quickTimeout(); //if we don't await again, the store has not been updated yet.
-    expect(chefSelectors.selectAll(store.getState())).toEqual(expectedResultForLookup);
+    expect(chefSelectors.selectAll(store.getState())).toEqual(
+      expectedResultForLookup
+    );
   });
 
-  it("fetchChefsById should look up CAPI details for profile tags", async ()=>{
+  it('fetchChefsById should look up CAPI details for profile tags', async () => {
     const store = createStoreAndFetchMock([
       {
-        pattern: "/api/live/tags?type=contributor&ids=profile%2Fanna-jones%2Cprofile%2Fanna-del-conte&show-elements=image&show-fields=all",
-        response: tagLookupResponse
-      }
+        pattern:
+          '/api/live/tags?type=contributor&ids=profile%2Fanna-jones%2Cprofile%2Fanna-del-conte&show-elements=image&show-fields=all&page-size=100',
```

Comment:
Another change for this PR: included pageSize.
Others are lint errors that got fixed by INtellJ.